.pre-commit-config.yaml

```diff
@@ -10,6 +10,7 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies: [pep8-naming, flake8-bugbear]
+        args: ['--classmethod-decorators=classmethod,validator,root_validator']
```

Comment:
out of curiosity, what does this do?

Reply:
Define what to be considered [as class method in `flake8`](https://github.com/PyCQA/pep8-naming/issues/38), making not mandatory to add `#noqa: N805` in front of each validator in pydantic (some are removed in this PR). By default, flake8 considers that the first argument has to be `self`, except if decorated with `classmethod`. This argument add `validator` and `root_validator` to this list.